js-client-sdk.api.md

```diff
@@ -25,7 +25,7 @@ export interface IAssignmentLogger {
 // @public
 export interface IClientConfig {
     apiKey: string;
-    assignmentLogger?: IAssignmentLogger;

```

Comment:
Made logger not optional; discussed with @vpai about doing that anyway to force consumers to think about logging.